Fhi.HelseId.Integration.Tests/Setup/TokenCreator.cs

```diff
@@ -58,13 +58,14 @@ private static async Task<string> RequestHelseIdTttToken()
         var auth = new AuthenticationService(config, apiConfig);
         await auth.SetupToken();
 
-        if (string.IsNullOrEmpty(auth.AccessToken))
+        var jwtAccessToken = auth.GetAccessToken(HttpMethod.Get, apiConfig.Url);
+        if (string.IsNullOrEmpty(jwtAccessToken.AccessToken))
```

Comment:
💭 
`AccessToken `kan aldri være null.
Skal vi bare sjekke `jwtAccessToken.AccessToken == string.Empty`? 🤔

Reply:
For better safety and readability, string.IsNullOrEmpty() is generally preferred,